### DISCUSSION TITLE
This application failed to start because no Qt platform plugin could be initialized.

### DISCUSSION THREAD

**Discussion by huskier** [MeVisLab/pythonqt#204] [Q&A] [answered]
After compiled the PythonQt, we try to run the example program PyGuiExample.exe in the lib directory, and we've gotten the following error. What's wrong with our program? Thanks in advance.
![ErrorMessage](https://github.com/MeVisLab/pythonqt/assets/5828575/3ca2843c-dc8f-4a87-83c4-61e3fe9d6d23)

**Comment by mrbean-bremen** [comment]
This looks like your Qt installation is not found, or the wrong one is found. Have you checked that the `plugins/platforms` directory exists in the path of the Qt binaries, and that the Qt binaries are in the PATH?

**Comment by huskier** [comment, marked as answer]
@mrbean-bremen Thank you. 
After added the user environment variable QT_QPA_PLATFORM_PLUGIN_PATH (value: C:\Application\Qt\5.15.2\msvc2019_64\plugins\platforms\), the issue is fixed.

**Comment by huskier** [comment]
Sometimes the variable QT_QPA_PLATFORM_PLUGIN_PATH can make the QT application start significant slowly. I've encountered the slow problem on my PC, and finally I've located the problem is the variable QT_QPA_PLATFORM_PLUGIN_PATH. After delete the variable, my application could just fast as before.
